torchmultimodal/modules/layers/quantisation.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, num_embeddings: int, embedding_dim: int):
             -1 / self.num_embeddings, 1 / self.num_embeddings
         )
         self.quantised_vectors = None
+        self.codebook_indices = None
 
     def _preprocess(self, x: Tensor):
```

Comment:
missing return type. any idea why mypy isnt catching these @ebsmothers ?

Reply:
@ankitade I think we need to set some of [these](https://mypy.readthedocs.io/en/stable/command_line.html#untyped-definitions-and-calls) flags in our mypy config file. Let me add them in a separate PR